pkg/apis/servicecatalog/types.go

```diff
@@ -384,6 +384,16 @@ type CommonServicePlanSpec struct {
 	// ServiceBindingCreateParameterSchema is the schema for the parameters that
 	// may be supplied binding to an ServiceInstance on this plan.
 	ServiceBindingCreateParameterSchema *runtime.RawExtension
+
+	// Currently, this field is ALPHA: it may change or disappear at any time
+	// and its data will not be migrated.when a bind operation stored in the Secret when binding to a ServiceInstance on this plan.
+	//
+	// ServiceBindingCreateResponseSchema is the schema for the response that
+	// will be returned by the broker when binding to a ServiceInstance on this plan.
+	// The schema also contains the sub-schema for the credentials part of the
+	// broker's response, which allows clients to see what the credentials
+	// will look like even before the binding operation is performed.
+	ServiceBindingCreateResponseSchema *runtime.RawExtension
```

Comment:
We should have a feature gate for this, since it's alpha.

Reply:
Done.